src/main/java/org/threadly/concurrent/PriorityScheduler.java

```diff
@@ -608,6 +621,26 @@ public void start(QueueManager queueManager) {
       }
       
       this.queueManager = queueManager;
+      
```

Comment:
Let me know if you have an ideas of a less hacky way to fix this.  I know we both recognized it as a small issue with the 4.4.0 release (and thus the TODO note that I have no removed from workerIdle).  Despite it being very small and not likely noticeable or a problem for most people I wanted to go ahead and improve it since we were doing a patch release.  Though somewhat hacky and confusing (thus all the comments) this seems like the most performant way to solve it.